lib/grpc/server/stream.ex

```diff
@@ -1,4 +1,4 @@
-defmodule GRPC.Server.Stream do
+defmodule GRPC.Server.Materializer do
```

Comment:
would this break packages like OTEL?

Reply:
This is definitely a break change from the point of view of the library version. But I don't think it would break OTEL, but we have to check.